BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -253,6 +254,11 @@ extension GiniBankNetworkingScreenApiCoordinator {
         }
     }
 
+    private func sendAnalyticsEventSDKClose() {
+        AnalyticsManager.track(event: .sdkClosed,
+                               properties: [AnalyticsProperty(key: .status, value: "successful")])
```

Comment:
[Clarification] do we need to send somewhere event with `unsuccessful`?

Reply:
@mrkulik for now no, but we want to easily add if we see that we need it.